packages/core/src/IconContainer/IconContainer.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { faAtom } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Meta, StoryObj } from "@storybook/react";
+import { HvIconContainer } from "@hitachivantara/uikit-react-icons";
+
+const meta: Meta<typeof HvIconContainer> = {
+  title: "Components/Icon Container",
+  component: HvIconContainer,
+};
+export default meta;
+
+export const Main: StoryObj<typeof HvIconContainer> = {
+  parameters: {
+    chromatic: { disableSnapshot: false },
+  },
+  args: {
+    size: undefined,
+  },
+  argTypes: {
+    size: { control: { type: "select" } },
```

Comment:
This is not working, and it'd be nice if it did. I think we should leave it as the default input, so that we can pass in a `number` too

![image](https://github.com/user-attachments/assets/3707be8b-9360-4a9b-b727-324e8c58755e)